packages/forge2d/lib/src/dynamics/world.dart

```diff
@@ -149,24 +149,24 @@ class World {
     bodies.remove(body);
   }
 
-  /// Create a joint to constrain bodies together. No reference to the definition is retained.
+  /// Adds a joint to constrain bodies together.
+  ///
   /// This may cause the connected bodies to cease colliding.
   ///
-  /// @warning This function is locked during callbacks.
-  T createJoint<T extends Joint>(JointDef def) {
+  /// Adding a joint doesn't wake up the bodies.
+  ///
+  /// Warning: This function is locked during callbacks.
+  void createJoint(Joint joint) {
```

Comment:
@spydon shall we return the `Joint` over `void`?

Reply:
I think we can skip that for now.